packages/flutter/lib/src/material/slider_theme.dart

```diff
@@ -284,13 +265,7 @@ class SliderThemeData extends Diagnosticable {
     const int thumbAlpha = 0xff;
     const int disabledThumbAlpha = 0x52; // 32% opacity
     const int valueIndicatorAlpha = 0xff;
-
-    // TODO(gspencer): We don't really follow the spec here for overlays.
-    // The spec says to use 16% opacity for drawing over light material,
-    // and 32% for colored material, but we don't really have a way to
-    // know what the underlying color is, so there's no easy way to
-    // implement this. Choosing the "light" version for now.
-    const int overlayLightAlpha = 0x29; // 16% opacity
+    const int overlayAlpha = 0x1f; // 12% opacity
```

Comment:
Move up one line so it's kind alpha.

Reply:
I moved it up but not sure how it fits into the overall organization of these alpha values.